lms/envs/common.py

```diff
@@ -515,6 +515,17 @@
         'OPTIONS': {
             'name': 'tracking'
         }
+    },
+    'oli': {
+        'ENGINE': 'track.backends.oli.OLIAnalyticsBackend',
+        'OPTIONS': {
+            'oli_analytics_service_url': 'https://cb-jorr-4.appspot.com/oli_target/rest/modules/learning_analytics/',
+            'path': 'event',
+            'key': 'learning-analytics',
+            'secret': 'correct horse battery staple',
```

Comment:
should the secret be empty here and be overwritten with loading the property from the 'secrets' repo?